kayenta-integration-tests/kayenta-integration-tests.gradle

```diff
@@ -16,8 +16,8 @@ test.testLogging {
 
 gradle.taskGraph.whenReady {
     tasks.test.enabled = (
-      properties.getOrDefault('ENABLE_INTEGRATION_TESTS', 'false') == 'true'
```

Comment:
Ah I see. [github action build.yml:32](https://github.com/spinnaker/kayenta/pull/708/files#diff-898e737ee4e56ba042bda31764bed49eL32) and [github action pr.yml:27](https://github.com/spinnaker/kayenta/pull/708/files#diff-d9db351496552ae42395e3057e47624eL27) was removed, and with the old logic, it would have never ran.